src/components/page/MembersPage.tsx

```diff
@@ -8,10 +10,28 @@ interface IMembersPageProps {
 export const MembersPage = async ({ id }: IMembersPageProps) => {
   const users: IUser[] = await fetchUserList(id);
   return (
-    <>
-      {users.map((user, i) => (
-        <div key={i}> {user.name} </div>
-      ))}
-    </>
+    <div className="px-4 pt-6 pb-16">
+      <div className="flex items-center justify-between pb-2">
+        <h2 className="text-xl">Members</h2>
+        <div className="flex gap-0.5">
+          <div className="flex justify-center items-center">
```

Comment:
This `<div>` could be replaced with `<Button>` instead, although it could also be done in the future update when implementing its functionality.

Another problem here is that the button size is controlled by its child. However, in this design, there are two buttons side by side, and we want to guarantee that they are exactly the same size. So, the better approach is to give width and height to the `<Button>`s which wrap the `<Icon>`s.

Reply:
Hi, Yusuke!
Thanks for additional comment and suggestion, I replaced <div> with <Button> and set width and height as you suggested.
Please check it out when you have time.
Thanks!